cmd/vicadmin/vicadm.go

```diff
@@ -472,6 +472,26 @@ func findDatastoreLogs(c *session.Session) (map[string]entryReader, error) {
 
 			log.Debugf("Added log file for collection: %s", logfile.URL.String())
 		}
+
+		f := logfile.VMName
+		if !c.IsVC() {
+			f = logfile.URL.Path
+		}
+
+		// get the vmx and vmsd files
+		for _, ext := range []string{"vmx", "vmsd"} {
```

Comment:
Just sharing my thoughts, this comment might is arguable and doesn't really worth any effort in this particular case to address it: From testability standpoint I would use a separate function and would call this function twice with different file extensions. From simplicity standpoint, I would probably avoid using loop and created two reader objects, but that would be questionable if needed to be extended later.